<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfflineLocalisationSettingsDialog</class>
 <widget class="QDialog" name="OfflineLocalisationSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="PruneMethodGroupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Prune Method</string>
   </property>
   <widget class="QComboBox" name="PruneMethodComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Merge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Max Likelyhood</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Viterbi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>N-Scan</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="PruneMethodParamLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N = </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="PruneMethodParamSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>57</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="BranchMethodGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Branch Method</string>
   </property>
   <widget class="QComboBox" name="BranchMethodComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Exhaustive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Selective</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Constraint</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Probabalistic</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FilteTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Filter Type</string>
   </property>
   <widget class="QComboBox" name="FilterTypeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Basic UKF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sequential UKF</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OfflineLocalisationSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OfflineLocalisationSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
